src/ui/views/Swap/hooks/token.tsx

```diff
@@ -505,7 +505,9 @@ export const useTokenPair = (userAddress: string) => {
         try {
           const suggestSlippage = await wallet.openapi.suggestSlippage({
             chain_id: findChainByEnum(chain)!.serverId,
-            slippage: Number(slippageObj.slippage || '0.1') / 100 + '',
+            slippage:
+              new BigNumber(slippageObj.slippage || '0.1').div(100).toFixed() +
+              '',
```

Comment:
```suggestion
              new BigNumber(slippageObj.slippage || '0.1').div(100).toFixed(),
```
`toFixed` 返回的本身就是个  string